github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1885,3 +1885,39 @@ periodics:
         privileged: true
     nodeSelector:
       type: bare-metal-external
+- name: periodic-kubevirt-image-bump
+  cron: "5 7 * * 6"
+  max_concurrency: 1
+  annotations:
+    testgrid-create-test-group: "false"
+  labels:
+    preset-gcs-credentials: "true"
+    preset-github-credentials: "true"
+    preset-bazel-unnested: "true"
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+  - base_ref: main
+    org: kubevirt
+    repo: kubevirt
+    workdir: true
+  - base_ref: main
+    org: kubevirt
+    repo: project-infra
+  cluster: ibm-prow-jobs
+  spec:
+    securityContext:
+      runAsUser: 0
+    containers:
+    - image: quay.io/kubevirtci/pr-creator:v20220927-5a02fb7
+      command: ["/bin/sh"]
+      args:
+      - "-ce"
+      - |
+        make bump-images
+        git-pr.sh -c "cd ../project-infra && bazelisk run //robots/cmd/uploader:uploader -- -workspace ${PWD}/../kubevirt/WORKSPACE -dry-run=false" -p ${PWD} -r kubevirt -L approved,release-note-none -T main
```

Comment:
Hm, I think this is alrady pretty good, however we now potentially create the same PR description like the uploader job does, so they would try to thake the PR over of each other.

I see two options:

* Run the bazel upload command before and give the PR a dedicated title, or
* Merge the two jobs, but if an updated introduces issues we may be blocked with automation, while the uploader/morror job itself would still work (and keep the builds working due to mirroring current referenced dependencies)

Reply:
I'd rather go with the first option. This way we have it separate and independent.

Reply:
sgtm.